ui/packages/consul-ui/app/templates/dc/services/instance.hbs

```diff
@@ -58,6 +58,9 @@ as |route|>
             </h1>
             <Consul::ExternalSource @item={{item}} />
             <Consul::Kind @item={{item}} @withInfo={{true}} />
+            {{#if proxy.ServiceProxy.Mode}}
```

Comment:
Does any value for `proxy.ServiceProxy.Mode` mean TransparentProxy? Or just truthy? I think you mentioned also, but don't forget to add this to the mocks I don't think I see that here.

Reply:
Mode is true/false. This `Mode` attribute is a new addition to the /connect endpoint. I've added that to the PR notes.

Reply:
I just spoke to @freddygv 

> Mode can be "", "direct", or "transparent"